MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -71,7 +71,7 @@ public abstract class BaseStreamingService : BaseApiService
         protected IMediaSourceManager MediaSourceManager { get; private set; }
         protected IJsonSerializer JsonSerializer { get; private set; }
 
-        public static IHttpClient HttpClient;
+        public static IHttpClient HttpClient { get; private set; }
```

Comment:
Doesn't really make sense as a static anymore

Reply:
Ah, good catch, should make it protected from what I can tell. Maybe I should move it into the BaseProgressiveStreamingService class? Would reduce the number of classes that need a HttpClient in their constructor.